### ISSUE TITLE
Will this work with lazysizes?

### ISSUE THREAD

**Issue by tettoffensive** [geeogi/nuxt-responsive-loader#12]
Wondering if this is setup to work with lazysizes - which uses `data-srcset` instead of `srcset`?

https://github.com/aFarkas/lazysizes

**Comment by geeogi** [comment]
@tettoffensive yep, should do. You just need to specify the correct import and webpack will do the hard work e.g. 
```
:data-srcset="require('~/assets/image.jpg').srcSet"
```

**Comment by tettoffensive** [comment]
@geeogi Cool that worked. Although I had an issue since I had a conflict in which I was also doing something with the image loader to remove svg's and load them inline with the html-loader. I just turned that into a plugin and inserted it after yours leaving only the gif alone.

I also ended up making a component so that I could do `vue.transformAssetUrls['lazy-image'] = 'source';` which means I no longer have to type `require` everywhere. I just have my component with `source="~assets/image.jpg"` and the responsive loader and lazysizes do the rest.
